src/app/profiles/store/actions/popin.ts

```diff
@@ -0,0 +1,13 @@
+import { ContextPopinState } from '../reducers/popin.reducer';
+import store from '../index';
+
+export const SET_POPIN = 'SET_POPIN';
+
+export const setPopin = (payload: ContextPopinState) => ({
```

Comment:
Usually we avoid magic setter action, but I don't mind here.
However it should name accordingly an use the right interfaces.
```
export const setPopinState = (popinState: ContextPopinState): StandardAction => ({
...
```

Every action should be a  `StandardAction` see other actions for further references.

Also I don't know if it's always a `ContextPopin` or only a `Popin`, sometimes you're using and sometimes the other. You should be consistent.